pkg/common/logging/api/loggerapi.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
is the api subdirectory needed? is there a more descriptive name for the package?

Reply:
renamed 'common/logging' to 'common/log', and moved logger interfaces right inside 'common/log'.